app/services/stripe_service/api/payments.rb

```diff
@@ -129,6 +135,8 @@ def payment_details(tx)
       end
 
       def payout(tx)
+        report = StripeService::Report.new(tx: tx)
+        report.capture_charge_start
```

Comment:
This report is in the wrong method. report.capture_charge should be used in the `capture` method above. Here the call is for creating payout. Add also reporting for creating payout in the `Report` class.

For payouts we should log:

before:

```json
{
  "stripe_op": "create_payout", 
  "event": "stripe_call",
  "transaction_id": 1234,
  "stripe_payment_id": 123,
  "stripe_seller_id": "acc_foobar",
  "stripe_charge_id": "ch_foobar"
}
```

after (success):

```json
{
  "stripe_op": "create_payout",
  "event": "stripe_call_succeeded",
  "transaction_id": 1234,
  "stripe_payment_id": 123,
  "stripe_seller_id": "acc_foobar",
  "stripe_charge_id": "ch_foobar",
  "stripe_payout_id": "po_foobar"
}
```

after (failure):

```json
{
  "stripe_op": "create_payout",
  "event": "stripe_call_failed",
  "transaction_id": 1234,
  "stripe_payment_id": 123,
  "stripe_seller_id": "acc_foobar",
  "stripe_charge_id": "ch_foobar",
  "stripe_error": {
    "message": "the 'message' from Stripe API response, if any",
    "code": "the code in stripe api response if any",
    "decline_code": "the decline_code if any"
  }
}
```